Week2/app.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const mysql = require('mysql');
+const process = require('process')
+const inquirer = require('inquirer');
+const util = require('util');
+const test = require('./importData.js');
+const prompt = require('prompt');
+
+
+    // 1- create connection 
+const db =  mysql.createConnection({
+    host     : 'localhost',
+    user     : 'root',
+    password : 'mosleh1234'
+});
+
+
+// 2- connect 
+
+    db.connect((err) => {
+    if(err) {
+        throw err;
+    }
+    // console.log('Mysql connected');
+});
+3- Create database 
+let sql = 'CREATE DATABASE IF NOT EXISTS new_world';
+ db.query(sql, (err) =>{
+   if(err) throw err;
+})
+// 4- use database new_world
+ db.query('USE new_world;', err => {
+    if (err) throw err;
+  });
+
+// const queryPromise = await util.promisify(db.query.bind(db));
+
+  const queryPromise =  util.promisify(db.query.bind(db));
+  const [_, __, ...question] = process.argv;
+ 
+  async function main(){
+      
+//   1- What is the capital of country X ? (Accept X from user)   
+        if (question[0] === 'capital') {
+            const input1 = await inquirer.prompt([
+            { name: 'Country', message: 'Insert Country:' },
+                ]);
+                const sql1 = `SELECT city.Name FROM country JOIN city on city.ID = country.Capital WHERE country.Name = ?`;
+            
+                let result1 = await queryPromise(sql1, input1.Country);
+                   if (result1[0] === undefined) {
+                    
+                    console.log(`${input1.Country} is not exist in database, please insert a valid value`);   
+                } 
+                else {
+                    console.log(`The capital of ${input1.Country} is: ${result1[0].Name}`);
+        } 
+   
+
+// 2- List all the languages spoken in the region Y (Accept Y from user)
+     } else if(question[0] === 'languages'){
+        const input2 = await inquirer.prompt([
+            { name: 'Region', message: 'Insert region:' },
+             ]);
+             const sql2 = `SELECT distinct Language FROM country join countrylanguage on country.Code=countrylanguage.CountryCode where Region =?`;
+             let result2 = await queryPromise(sql2, input2.Region);
+             if (result2[0]=== undefined) {
+                 console.log(`${input2.Region} is not exist in database, please insert a valid value`);
+                 
+             } else {
+                 console.log(`The spoken languages in ${input2.Region} are:`);
+                 console.table(result2)
```

Comment:
console.table doesn't print anything I don't know if it's a problem of node js version, does it work for you? I changed them all to log

Reply:
Yes, actually it works well with me. however, I have changed them to log :)